mautic/mautic
# Add maintenance result to audit log

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for all features, enhancements and bug fixes (until 3.3.0 is released) 
| Bug fix?                               | yes/no
| New feature?                           | yes/no
| Deprecations?                          | yes/no
| BC breaks?                             | yes/no
| Automated tests included?              | yes/no
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 




#### Description:
Maintenance command clean old data from Mautic https://docs.mautic.org/en/setup/cron-jobs#clean-up-old-data
With --gdpr flag remove also contacts without activity older like 3 years.
We need summary from maintenance command store to audit log to be sure contact's were deleted from command.



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Be sure you you have contacts with last_active  older like 3 years ago
3. `run php bin/console  mautic:maintenance:cleanup --gdpr`
4. Go to audit_log table and see new log  with of summary of command

![image](https://user-images.githubusercontent.com/462477/130347755-94061f19-a1a6-4e0f-b7c3-e4fc5b118906.png)

`a:2:{s:7:"options";a:3:{s:8:"days-old";i:365;s:4:"gdpr";b:1;s:3:"env";s:3:"dev";}s:5:"stats";a:1:{s:8:"Visitors";i:1;}}`




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10351"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
